js/jquery.mapael.js

```diff
@@ -1469,6 +1467,18 @@
                 elem.mapElem.toFront();
             }
 
+            // Set the cursor attribute related to the HTML link
+            if (elem.options.href !== undefined) {
+                elem.options.attrs.cursor = "pointer";
+                if (elem.options.text) elem.options.text.attrs.cursor = "pointer";
+            } else {
+                // No HTML links, check if a cursor was defined to pointer
+                if (elem.mapElem.attrs.cursor === 'pointer') {
```

Comment:
Why did you add this check before setting the cursor style to 'auto' ?

Reply:
In case the user did set the cursor to something other than pointer (like `help`...)...

Reply:
In order to avoid overriding the user attribute.

Reply:
Ok thanks, I got it ! It will not work for the users who willingly set cursor to 'pointer', but it should be a very unusual case, and its still better than before :)